test/tests/api/v2_0/skupack_tests.py

```diff
@@ -0,0 +1,422 @@
+                    result = self.__client.last_response
+                    data = loads(self.__client.last_response.data)
+                    sku_id = data['id']
+                    LOG.info("ID of the posted sku is:   "+ sku_id)
+                    time.sleep(3)
+                    assert_equal(201, result.status, message=result.reason)
+                    LOG.info("node_id " + node_id)
+                    #Validate that the sku element in the node has been updated with right sku ID
+                    Api().nodes_get_by_id(identifier=node_id)
+                    updated_node = loads(self.__client.last_response.data)
+                    LOG.info("updated_noded is : {0}".format(updated_node))
+                    arr =updated_node['sku'].split("/")
+                    assert_equal(sku_id, arr[4])
+                    #validate the /api/2.0/skus/:id/nodes works
+                    Api().skus_id_get_nodes(sku_id)
+                    result = self.__client.last_response
+                    data = loads(self.__client.last_response.data)
+                    assert_equal(200, result.status, message=result.reason)
+                    flag = False
+                    for item in data:
+                        if item["id"] ==  node_id:
+                            flag = True
+                            break
+                    assert_true(flag)
+
+                    #delete the sku that where created
+                    LOG.info("Deleting the added sku")
+                    Api().skus_id_delete(identifier=sku_id)
+                    result = self.__client.last_response
+                    assert_equal(200, result.status, message=result.reason)
+            i = i +1
+
+    @test(groups=['skus_api2.tests', 'api2_post_skupack'], depends_on_groups=['api2_get_sku_nodes'])
+    def post_skupack(self):
+        """Test POST:api/2.0/skus/pack"""
+        for n in self.__nodes:
+            if n.get('type') == 'compute':
+                # update the sku rule above (rules[0].name.contains) with a value from the cataloged node
+                node_id = n.get('id')
+                Api().nodes_get_catalog_by_id(identifier=node_id)
+                node_catalog_data = loads(self.__client.last_response.data)
+                if len(node_catalog_data) > 0:
+                    node_manufacturer = (node_catalog_data)[0].get('data').get("System Information").get("Manufacturer").split(" ")[0]
+
+
+                    #Post the sku pack
+                    self.generateTarball(node_manufacturer)
+                    self.__file = {'file': open(self.__skuPackTarball, 'rb')}
+                    URL = config.host + '/skus/pack'
+                    LOG.info("URL {0}".format(URL))
+                    LOG.info(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
+                    LOG.info(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
+                    LOG.info(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
+                    LOG.info(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
```

Comment:
Ahh, maybe this is just your debug code, not so suitable to merge into the master branch.